cost-analyzer/templates/forecasting-deployment.yaml

```diff
@@ -0,0 +1,134 @@
+{{- if .Values.forecasting.enabled }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "forecasting.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "forecasting.commonLabels" . | nindent 4 }}
+    {{- with .Values.global.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{ include "forecasting.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: Recreate
```

Comment:
cost-analyzer defaults to `RollingUpdate`. Are we sure we want `Recreate` here?

Reply:
Switched to RollingUpdate for consistency with other Deployments, I'm not sure why Recreate was chosen initially.